src/Configuration/ConfigurationRuleset.php

```diff
@@ -25,6 +29,8 @@ public static function fromArray(array $arr): self
     private function __construct(array $layerMap)
     {
         $this->layerMap = $layerMap;
+        $this->skipViolations = ConfigurationSkippedViolation::fromArray($layerMap['skip_violations'] ?? []);
```

Comment:
Would it make sense to you to collapse `ConfigurationSkippedViolation` into `ConfigurationRuleset` here?

Reply:
Refactored.